server/src/main/java/com/linecorp/line/auth/fido/fido2/server/exception/FIDO2ServerRuntimeException.java

```diff
@@ -62,4 +62,8 @@ public static FIDO2ServerRuntimeException makeCredNotFoundUser(String rpId, Stri
                 "RpId: " + rpId + "; UserId: " + userId);
     }
 
+    public static FIDO2ServerRuntimeException makeSignatureVerificationError(String aaguid) {
+        throw new FIDO2ServerRuntimeException(InternalErrorCode.ASSERTION_SIGNATURE_VERIFICATION_FAIL,
+                "AAGUId: " + aaguid);
```

Comment:
Adding aaguid in exception message may be hard to use, unless user knows about the implementation detail that
- this is due to `ASSERTION_SIGNATURE_VERIFICATION_FAIL`
- aaguid can be parsed from the exception message

I would prefer adding a nullable field for aaguid in `FIDO2ServerRuntimeException` so that user can directly get it and customize error response when needed.

What do you think?